Fixed content. (#1352)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/intro-engine.adoc

Mule Event processors indicate to Mule whether they are CPU intensive, CPU light, or IO intensive operations. These workload types help Mule tune for different workloads, so you don’t need to manage thread pools manually to achieve optimum performance. Instead, Mule introspects the available resources (such as memory and CPU cores) in the system to tune thread pools automatically.

See xref:execution-engine.adoc[Execution Engine] for complete details about the new threading model and upgrade instructions. 

== Upgrading from Mule 4.2/4.1

If you are upgrading from Mule 4.2.x or 4.1.x to Mule 4.3.x, take the following actions:

* If there are no custom threading settings applied (either through the `scheduler-pools.conf` file or directly in the Mule app), then no action is required.
* If there are custom threading configurations applied, test your Mule applications with the default configuration. +
Because of the UBER pool strategy and other performance updates implemented in Mule 4.3, you might not need custom threading configurations.

> * If tests confirm that custom configuration is still necessary, then reach out to our Support team. +

Reviewer: ```suggestion
* If tests confirm that custom configuration is still necessary, contact our Support team. +
```
